helm-chart/splunk-kubernetes-metrics/values.yaml

```diff
@@ -176,3 +176,38 @@ kubernetes:
   clusterName:
   # Add privileged access to containers for openshift compatability
   openshift: false
+
+  # `customFilters` defines the custom filters to be used.
+  #
+  # The scheme to define a custom filter is:
+  #
+  # ```
+  # <name>:
+  #   tag: <fluentd tag for the filter>
+  #   type: <fluentd filter type>
+  #   body: <definition of the fluentd filter>
+  # ```
+  #
+  # = fluentd tag for the filter =
+  # This is the fluentd tag for the record
+  #
+  # = fluentd filter type =
+  # This is the fluentd filter that the user wants to use for record manipulation.
```

Comment:
Are there any other supported filters outside of `jq_transformer` - If so we should link out or document. Or explicitly say only supported/tested for `jq_transformer`